FHEM/14_SD_WS_Maverick.pm

```diff
@@ -344,6 +334,7 @@ sub SD_WS_Maverick_updateReadings($){
   <b>Known models:</b>
   <ul>
     <li>Maverick 732/733</li>
+    <li>TFA 14.1504 (Küchen-Chef Funk-Bratenthermometer)</li>
```

Comment:
Eine englischen Namen gibt es vermutlich nicht und jetzt übersetzen ist auch blöde :(

Reply:
Ich habe mich da an die Bezeichnung des Herstellers gehalten. Auf der [Produktseite](https://www.tfa-dostmann.de/produkt/funk-grill-bratenthermometer-kuechen-chef-14-1504/) gibt es eine Anleitung, in der als englische Übersetzung _Radio-controlled grill and meat thermometer_ angegeben wird - wäre DIr das lieber?